README.md

```diff
@@ -32,6 +32,33 @@ tags:
 Full job description here, in Markdown format
 ```
 
+This is an example of how it might look when filled in. When the document is complete there should be no more brackets and the comment after the timestamp should be removed.
+
+
+```
+---
+title: Intern Django Developer
+company: Sample Company
+url: http://samplecompany.co.uk
+location: Burton on Trent, Staffordshire, England
+contract: permanent
+contact:
+    name: Mrs A. B. Sample
+    email: absample@momsbikes.co.uk
+    phone: +44 0208 123456
+    ...: ...
```

Comment:
Yeah - perhaps I will write something more serious

On Tue, 7 Jul 2015 08:16 Daniel Pope <notifications@github.com> wrote:

> In README.md
> <https://github.com/pythonjobs/jobs/pull/91#discussion_r34012979>:
>
> > @@ -32,6 +32,33 @@ tags:
> >  Full job description here, in Markdown format
> >  ```
> >
> > +This is an example of how it might look when filled in. When the document is complete there should be no more brackets and the comment after the timestamp should be removed.
> > +
> > +
> > +```
> > +---
> > +title: Intern Django Developer
> > +company: Sample Company
> > +url: http://samplecompany.co.uk
> > +location: Burton on Trent, Staffordshire, England
> > +contract: permanent
> > +contact:
> > +    name: Mrs A. B. Sample
> > +    email: absample@momsbikes.co.uk
> > +    phone: +44 0208 123456
> > +    ...: ...
>
> If this is an example this kind of think should be omitted.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pythonjobs/jobs/pull/91/files#r34012979>.
>